dev/ci/ci-quickchick.sh

```diff
@@ -10,6 +10,6 @@ git_download quickchick
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/quickchick"
-  make
-  make install
+  dune build --root . -p coq-quickchick @install
```

Comment:
actually should be 
```suggestion
  dune build -p coq-quickchick @install
```
(-p includes -root .)
sorry for previous incorrect suggestion